internal/provider/resource_saml_group_mapping.go

```diff
@@ -0,0 +1,382 @@
+package provider
+
+import (
+	"context"
+	"errors"
+	"slices"
+	"strings"
+
+	"github.com/google/uuid"
+	"wiz.io/hashicorp/terraform-provider-wiz/internal/utils"
+
+	"github.com/hashicorp/terraform-plugin-log/tflog"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+
+	"wiz.io/hashicorp/terraform-provider-wiz/internal"
+	"wiz.io/hashicorp/terraform-provider-wiz/internal/client"
+	"wiz.io/hashicorp/terraform-provider-wiz/internal/wiz"
+)
+
+// ReadSAMLGroupMappings represents the structure of a SAML group mappings read operation.
+// It includes a SAMLGroupMappings object.
+type ReadSAMLGroupMappings struct {
+	SAMLGroupMappings SAMLGroupMappings `json:"samlIdentityProviderGroupMappings"`
+}
+
+// SAMLGroupMappings represents the structure of SAML group mappings.
+// It includes PageInfo and a list of Nodes.
+type SAMLGroupMappings struct {
```

Comment:
This should be a new type, and in the wiz/structs.go file: `SAMLIdentityProviderGroupMappingsConnection`